### ISSUE TITLE
chp. 2 connection refused

### ISSUE THREAD

**Issue by scollier** [dustymabe/summit-2017-container-lab#43]
```
[student@localhost bigapp]$ docker port bigapp 1080
0.0.0.0:32768
[student@localhost bigapp]$ curl -L http://cdk.example.com:32768
curl: (7) Failed connect to cdk.example.com:32768; Connection refused
```

**Comment by dustymabe** [comment]
actually this is an issue with the run command:

```
docker run -p 1080 --name=bigapp -e DBUSER=user -e DBPASS=mypassword -e DBNAME=mydb -d bigimg 
```

It should have -p 80 since 80 is the port the container is going to be listening on and  since we are letting "docker" assign a high port to map to port 80 we don't need to use 1080.

**Comment by dustymabe** [comment]
fixed in #49
